fhir-query-helper-base/src/main/java/io/elimu/a2d2/cds/fhir/helper/QueryingServerHelperBase.java

```diff
@@ -717,6 +717,17 @@ protected static void iRestfulClientFactoryConfig(IRestfulClientFactory factory)
 
 	public abstract FhirResponse<IBaseResource> fetchServer(final String resourceType, String resourceQuery);
 	
+	public FhirFuture<FhirResponse<IBaseResource>> fetchDirectAsync(String resourceType, String id, String asyncId) {
```

Comment:
Should we rename this method to `getResourceByIdInPathAsync`  to match method names used elsewhere in this file - for example, see a couple of methods above for `getResourceByIdAsync`.